### ISSUE TITLE
pdnsutil add-zone-key adds flag 256 CSK by default if no KSK|ZSK option is provided

### ISSUE THREAD

**Issue by ZaphodB** [PowerDNS/pdns#10372]
- Program: Authoritative
 - Issue type: Bug report

### Short description
If the ZSK|KSK option is not provided on pdnsutil add-zone-key it will generate a flag 256 CSK which is confusing because most registrars default to flag 257 when you add DNSSEC public keys|DS on their (web) interfaces.

### Environment
 - Operating system: Debian buster/10
 - Software version: 4.4.1-1pdns.buster
 - Software source: PowerDNS repository

### Steps to reproduce
1. pdnsutil create-zone example.com
2. pdnsutil add-zone-key example.com ECDSAP256SHA256
3. pdnsutil show-zone example.com

### Expected behaviour
Default behaviour of pdnsutil add-zone-key should match what secure-zone does, so adding a CSK with flag 257 instead of flag 256.

### Actual behaviour
```
root@shinjuku:~# pdnsutil create-zone example.com
Creating empty zone 'example.com'
root@shinjuku:~# pdnsutil show-zone example.com
This is a Native zone
Zone is not actively secured
Metadata items: None
No keys for zone 'example.com'.
root@shinjuku:~# pdnsutil show-zone example.com
This is a Native zone
Zone is not actively secured
Metadata items: None
Zone has NSEC semantics
keys: 
ID = 64 (CSK), flags = 256, tag = 62210, algo = 13, bits = 256  Inactive         Published  ( ECDSAP256SHA256 ) 
CSK DNSKEY = example.com. IN DNSKEY 256 3 13 vg0ZmSiHnVpKSG2dvYgbOKF/9AiR+iKfsHF77QQyS97RZwVvWj9uTU3YXwqGL9OL7SG8XAWaUVfa7R9bFRVEZQ== ; ( ECDSAP256SHA256 )
DS = example.com. IN DS 62210 13 1 27dcdb4192fb325b6b54e3cffc8c498892e97de9 ; ( SHA1 digest )
DS = example.com. IN DS 62210 13 2 14a224f00a41ee22d3114d4070f4774b81d18ce94aee3af545b3002240f28384 ; ( SHA256 digest )
DS = example.com. IN DS 62210 13 4 dd632c3df98249d3fb3e750f72871390577bc23aae36a1cb1e09a0ea9de51426a2634958db27f40a95a8877d70b422b2 ; ( SHA-384 digest )
```

vs.

```
root@shinjuku:~# pdnsutil remove-zone-key example.com 64
root@shinjuku:~# pdnsutil show-zone example.com
This is a Native zone
Zone is not actively secured
Metadata items: None
No keys for zone 'example.com'.
root@shinjuku:~# pdnsutil secure-zone example.com
Securing zone with default key size
Adding CSK (257) with algorithm ecdsa256
Zone example.com secured
Adding NSEC ordering information 
root@shinjuku:~# pdnsutil show-zone example.com
This is a Native zone
Metadata items: None
Zone has NSEC semantics
keys: 
ID = 65 (CSK), flags = 257, tag = 64921, algo = 13, bits = 256    Active         Published  ( ECDSAP256SHA256 ) 
CSK DNSKEY = example.com. IN DNSKEY 257 3 13 6UthEyCM1CWFFv41q2PUE3l494bO/1o77vRerD526vRjhWvpbI3ahhWw7aK2pQIxQkpVwAUlf3IEn1lM4gh3LA== ; ( ECDSAP256SHA256 )
DS = example.com. IN DS 64921 13 1 763c0a167e85220351cd6e5dc296b3c989fd2fb3 ; ( SHA1 digest )
DS = example.com. IN DS 64921 13 2 34ea98bf4d91ed13b6bdb57cfdc4c7304b3c882890d98bd9e28ad29feb5d77e5 ; ( SHA256 digest )
DS = example.com. IN DS 64921 13 4 1e4c67b44b3b670306c5d70c6dedb36fe6e60741b3210efcf867a773497b2f240b211b15c5b0ec1ed3bedc97fc809a75 ; ( SHA-384 digest )
```

**Comment by mnordhoff** [comment]
I had a related problem: I was trying to do a CSK rollover, I ran add-zone-key expecting it to create another 257 CSK/KSK, and it created a 256 ZSK instead.

**Comment by Habbie** [comment]
I checked all other pdnsutil methods that make or load keys, and they appear to have the right 257 default. I should also check the API.

**Comment by Habbie** [comment]
The API demands an explicit choice from the user, so that is fine too.

Fixed in #10373
